src/vs/workbench/contrib/notebook/common/model/notebookTextModel.ts

```diff
@@ -215,6 +225,11 @@ export class NotebookTextModel extends Disposable implements INotebookTextModel
 		return this._alternativeVersionId;
 	}
 
+	get notebookType() {
```

Comment:
Since we are already hardcoding `repl` to `jupyter-notebook`, why do we need `repl` view type? If `repl` view type is read in some components, this might lead to interesting bugs.